src/statsd_client.cpp

```diff
@@ -64,16 +64,8 @@ StatsdClient::StatsdClient(const std::string& host, const std::string& nodename,
         return;
     }
 
-    SOCKET hSocket = socket(AF_INET, SOCK_DGRAM, IPPROTO_UDP);
-    if (hSocket == INVALID_SOCKET) {
-        LogPrintf("Cannot create socket (socket() returned error %s), cannot init StatsdClient\n",
-                  NetworkErrorString(WSAGetLastError()));
-        return;
-    }
-    m_sock = std::make_unique<Sock>(hSocket);
-
     CNetAddr netaddr;
-    if (!LookupHost(m_host, netaddr, /*fAllowLookup=*/true) || !netaddr.IsIPv4()) {
+    if (!LookupHost(m_host, netaddr, /*fAllowLookup=*/false) || !(netaddr.IsIPv4() || netaddr.IsIPv6())) {
```

Comment:
nit: separate error message for ipv4 / ipv6 not

Reply:
Resolved in latest push!